python/pyarrow/tests/test_flight.py

```diff
@@ -54,6 +57,8 @@
 # Marks all of the tests in this module
 # Ignore these with pytest ... -m 'not flight'
 pytestmark = pytest.mark.flight
+# All tests on this module can be run when numpy is or is not available
+pytestmark = pytest.mark.without_numpy
```

Comment:
But does this mean they wouldn't be run if NumPy is available? It is not clear to me as a reader of the code, and we risk ignoring tests by mistake.